app/src/main/java/com/concordia/mcga/lib/BuildingBottomSheetInfo.java

```diff
@@ -0,0 +1,1168 @@
+package com.concordia.mcga.lib;
```

Comment:
There never was a copyright.
I commented every method, You can read the javadoc
BAsically to use the bottomsheet library you need to extend it with a class and overrite methods... Just like an OnClickListener